src/components/PledgedCollectiveCard.js

```diff
@@ -0,0 +1,223 @@
+`;
+
+const CollectiveNameText = styled(CardText)`
+  min-height: 20px;
+  font-size: 14px;
+  margin: 5px;
+  font-weight: 700;
+  text-align: center;
+  color: #303233;
+  white-space: nowrap;
+`;
+
+const CollectiveDescriptionText = styled(CardText)`
+  font-weight: bold;
+  text-align: center;
+  color: #787d80;
+  font-size: 1rem;
+  line-height: 1.3;
+  margin: 5px;
+  text-transform: uppercase;
+`;
+
+const CollectiveWebsiteText = styled(CardText)`
+  font-weight: normal;
+  text-align: center;
+  color: #0096f9;
+  font-size: 1.2rem;
+  line-height: 1.3;
+  margin: 0px;
+`;
+
+const MakePledgeButtonContainer = styled.div`
+  margin: 30px;
+  text-align: center;
+`;
+
+const ClaimCollectiveButtonContainer = styled.div`
+  margin: 30px;
+  text-align: center;
+`;
+
+const CardFooter = styled.div`
+  font-size: 1.1rem;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  border-top: 1px solid;
+  border-color: ${themeGet('colors.black.200')};
+`;
+
+const StatsContainer = styled.div`
+  display: flex;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  justify-content: space-around;
+`;
+
+const StatsItemContainer = styled.div`
+  height: 100%;
+  width: 50%;
+`;
+
+const StatsItemContainerWithBorder = styled(StatsItemContainer)`
+  border-left: 1px solid;
+  border-color: ${themeGet('colors.black.200')};
+`;
+
+const StatsItem = styled.div`
+  text-align: center;
+  margin: auto;
+`;
+
+const StatsItemValue = styled(StatsItem)`
+  font-weight: bold;
+  text-align: center;
+  color: #303233;
+  font-size: 1rem;
+  margin: 3px 2px 0px;
+`;
+
+const StatsItemLabel = styled(StatsItem)`
+  font-size: 1rem;
+  text-align: center;
+  font-weight: 300;
+  color: #303233;
+`;
+
+class PledgedCollectiveCard extends React.Component {
+  static propTypes = {
+    collective: PropTypes.object.isRequired,
+    membership: PropTypes.object,
+  };
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const { collective } = this.props;
+
+    const pledgeStats = collective.pledges.reduce(
+      (stats, { totalAmount }) => {
+        stats.backers++;
+        stats.totalAmount += totalAmount;
+        return stats;
+      },
+      {
+        totalAmount: 0,
+        backers: 0,
+      },
+    );
+
+    let website = collective.website;
+    if (!website && collective.githubHandle) {
+      website = `https://github.com/${collective.githubHandle}`;
+    }
+
+    return (
+      <StyledCard minWidth={215} maxWidth={215} height={380} borderRadius={8} bg="#ffffff">
+        <CardHeader>
+          <LogoBorder></LogoBorder>
+          <LogoContainer>
+            <Logo src={collective.logo} type={collective.type} website={collective.website} height={65} />
+          </LogoContainer>
+        </CardHeader>
+        <CardBody>
+          <CollectiveNameText>{collective.name}</CollectiveNameText>
+          <CollectiveDescriptionText title="Pledged Collective">Pledged Collective</CollectiveDescriptionText>
```

Comment:
As per https://github.com/opencollective/opencollective/issues/1387#issuecomment-469169313, the string `Pledged Collective` should be internationalized with `FormattedMessage`